docs/tutorials/alpa_on_slurm.rst

```diff
@@ -0,0 +1,320 @@
+Using Alpa on Slurm
+##########################
+
+This page provides instructions to run Alpa on clusters managed by Slurm.
+This guide is modified from `Deploy Ray on Slurm <https://docs.ray.io/en/latest/cluster/vms/user-guides/community/slurm.html>`_.
+
+Prerequisites
+**************
+
+Alpa requires CUDA and runs on GPU. Therefore, you need access to CUDA-compatible GPU in the cluster to run Alpa.
+
+Please make sure you have or can create an environment with required dependencies installed for Alpa.
+These dependencies are described in `Install Alpa <https://alpa.ai/install.html>`_.
+
+    .. note::
+
+        One easy way to create an environment with all dependencies installed is to create an environment using package management systems like `conda <https://docs.conda.io/en/latest/>`_ and setup the environment running an interactive job on Slurm.
```

Comment:
add a section to describe how to prepare an alpa python environment on slurm (assuming slurm has a Python virutalenv manager such as conda)